Formula/lib/libtatsu.rb

```diff
@@ -0,0 +1,37 @@
+class Libtatsu < Formula
+  desc "Library handling the communication with Apple's Tatsu Signing Server (TSS)"
+  homepage "https://libimobiledevice.org/"
+  url "https://github.com/libimobiledevice/libtatsu/releases/download/1.0.3/libtatsu-1.0.3.tar.bz2"
+  sha256 "4f69452d23e50e0ffbe844110e6ab6a900d080e051fbda3b7d595f679dee9bc5"
+  license "LGPL-2.1-or-later"
+
+  depends_on "autoconf" => :build
+  depends_on "automake" => :build
+  depends_on "libtool" => :build
+  depends_on "pkg-config" => :build
+  depends_on "libplist"
+
+  uses_from_macos "curl"
+
+  def install
+    if build.head?
+      system "./autogen.sh", *std_configure_args, "--disable-silent-rules"
+    else
+      system "./configure", *std_configure_args, "--disable-silent-rules"
+    end
```

Comment:
can you do like this?
```suggestion
    system "./autogen.sh" if build.head?
    system "./configure", "--disable-silent-rules", *std_configure_args
```

Reply:
This would lead to `./configure` being ran twice, as the provided `./autogen.sh` script, as with every `libimobiledevice` project, automatically executes it. I actually copy-pasted these lines without modification from the already present [libimobiledevice-glue.rb (L25-32)](https://github.com/Homebrew/homebrew-core/blob/master/Formula/lib/libimobiledevice-glue.rb#L25-L32).

Reply:
oh shoot, the configure is in autogen? maybe that should be removed in the upstream.